lib/snmp/test/exp/snmp_agent_bl_test.erl

```diff
@@ -4993,14 +4993,14 @@ otp_4394_config(AgentDir, MgrDir, Ip0) ->
     ?line write_community_conf(AgentDir, [C1, C2]),
     ?line update_vacm(Vsn, AgentDir),
     Ta1 = {"shelob v1", 
-	   [134,138,177,177], 5000, 1500, 3, %% Anv�nd Ip och modda
+	   [134,138,177,177], 5000, 1500, 3, %% Anv�nd Ip och modda
```

Comment:
Do you know what happened with swedish character here?

Reply:
If swedish is used in a comment please translate it to english at least if the line with the comment is touched in a commit.
We should not have swedish in the comments.

Reply:
It was unintended, I will drop accidental changes caused by improper editor that broke non-ASCII characters. I think it might be better to clean up comments in a separate patch for review convenience.